presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -211,7 +204,8 @@ public static QueryRunner createNativeQueryRunner(
                                 format("connector.name=tpch%n").getBytes());
 
                         // Disable stack trace capturing as some queries (using TRY) generate a lot of exceptions.
-                        return new ProcessBuilder(prestoServerPath, "--logtostderr=1", "--v=1")
+                        return new ProcessBuilder(Paths.get(prestoServerPath).toAbsolutePath().toString(),
```

Comment:
This looks like something independent of the new defaults. Please can you create another PR for it.

Reply:
This was updated to be consistent with the `ProcessBuilder` logic inside of `startRemoteFunctionServer` where it specifies the absolute path. We should do this in case a user specifies a relative path to their server binary.

Reply:
Thanks. That is reasonable. Can you separate this into its own PR ? Will help us isolate changes.